src/app/components/article-comparison/article-comparison.component.ts

```diff
@@ -38,24 +43,40 @@ export class ArticleComparisonComponent implements OnInit, OnChanges {
     // Flag to signal if the text-to-speech service is speaking.
     isSpeaking$: Observable<boolean>;
 
-    constructor(private store: Store<AppState>) {
+    // Base64 encoding of the user's recording.
+    recordingEncoding$: Observable<string>;
+
+    // Web server/Azure function endpoint for converting recording.
+    // TODO: Any better place for this to live?
+    convertApiUrl: any;
+
+    constructor(private store: Store<AppState>, private http: HttpClient) {
         this.articleLevelStatus$ = store.select(selectArticleLevelStatus);
         this.phraseNum$ = store.select(selectPhraseNum);
         this.phraseLevelStatus$ = store.select(selectPhraseLevelStatus);
         this.transcript$ = store.select(selectTranscript);
         this.isSpeaking$ = store.select(selectIsSpeaking);
+        this.recordingEncoding$ = store.select(selectRecordingEncoding);
+        // this.convertApiUrl = "http://127.0.0.1:5000/blob2mp3";
+        this.convertApiUrl = "https://conversiontest.azurewebsites.net/api/httptrigger1";
+        this.recordingEncoding$.subscribe((base64Encoding) => {
+            if (base64Encoding === undefined) {
+                return;
+            }
+
+            // TODO: send base64 encoding to Azure :)
+            this.http.post(this.convertApiUrl, { blob: base64Encoding }, { responseType: "text" }).subscribe((res) => {
+                console.log(res);
```

Comment:
I think we should notify the user (via the UI) when the audio recording has been processed.
But we can leave this feature for another UI story.

Reply:
Ah good idea! Taking note to include this in Azure Function #14